components/sysutils/virtualbox/Makefile

```diff
@@ -67,7 +68,6 @@ VBOX_FS_MODULES= vboxfs
 
 VBOX_BINS= VBoxControl VBoxService VBoxClient
 VBOX_SBINS= vboxmslnk
-VBOX_OGLLIBS= VBoxOGL.so VBoxEGL.so
```

Comment:
Perhaps, 11-fix-vbox_vendor_select.patch is no longer necessary, likely it's not needed as EGL libs has gone. The same stands for 12-disable-vbox-opengl.patch. Please, look them through.